runtime/src/accounts_db.rs

```diff
@@ -1308,13 +1328,18 @@ impl AccountsDB {
 
     fn hash_accounts(&self, slot_id: Slot, accounts: &[(&Pubkey, &Account)]) -> Vec<Hash> {
         let mut stats = BankHashStats::default();
+        let mut zla_count = 0;
         let hashes: Vec<_> = accounts
             .iter()
             .map(|(pubkey, account)| {
                 stats.update(account);
+                if account.lamports == 0 {
+                    zla_count += 1;
```

Comment:
I think this should be incremented only when rooted. Not critical, though.

Reply:
Yea.. that's true. Maybe need to re-think this.

Reply:
I have some interesting idea [here](https://github.com/solana-labs/solana/pull/8436#discussion_r383748726) :)